newlib/libc/tinystdio/vasnprintf.c

```diff
@@ -35,7 +35,7 @@
 
 #include "stdio_private.h"
 
-char *
+char * __disable_sanitizer
```

Comment:
Is this needed even with the POINTER_MINUS fix?

Reply:
Yeah, the sanitizer does not like computing the buffer start based on the end address and size. It complains about the resulting pointer being out of range. I couldn't figure out how to make it happy, so I just disabled it for functions dealing with `struct __file_str` bits.